src/data.ts

```diff
@@ -1,4 +1,4 @@
-const plurals: Record<string, string> = {
+const plurals_old: Record<string, string> = {
```

Comment:
Do you think we can remove this constant?

Reply:
yessssss. I forgot